keps/sig-node/3542-cri-image-pulling-with-progress-notification/kep.yaml

```
@@ -0,0 +1,31 @@
+title: CRI image pull progress
+kep-number: 3542
+authors:
+  - "@byako"
+  - "@jukkar"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: implementable
```

```suggestion
status: provisional
```
This will be easier to merge as a provisional design, and then we can iterate on it.

---

Thanks.